auto/cc/test

```diff
@@ -84,7 +84,9 @@ case $NXT_CC_NAME in
         #NXT_CFLAGS="$NXT_CFLAGS -fomit-frame-pointer"
         #NXT_CFLAGS="$NXT_CFLAGS -momit-leaf-frame-pointer"
 
-        # -Wstrict-overflow is supported by GCC 4.2+.
+        # While -Wstrict-overflow is supported by GCC 4.2+,
+        # at level 3 and above it is prone to give false
+        # positives.
         #NXT_CFLAGS="$NXT_CFLAGS -Wstrict-overflow=5"
```

Comment:
535107e8a131 ("Expand the comment about -Wstrict-overflow on GCC"):

```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

Reply:
Or maybe just remove the flag and the comment, and write the expanded comment in the git commit.  Why keep that unwanted flag around?